### PR TITLE
Setting for Binding Stateless Operations

### PR DESCRIPTION
**Pull request by nibanks**
Adds a settings knob that can be used to tune the timeout and max count for stateless operations on Bindings.

### CODE REVIEW

File: docs/api/QUIC_SETTINGS.md [hunk 1]
```diff
@@ -61,18 +67,24 @@ typedef struct QUIC_SETTINGS {
...
     uint8_t PacingEnabled                   : 1;
     uint8_t MigrationEnabled                : 1;
     uint8_t DatagramReceiveEnabled          : 1;
-    uint8_t ServerResumptionLevel           : 2;
+    uint8_t ServerResumptionLevel           : 2;    // QUIC_SERVER_RESUMPTION_LEVEL
     uint8_t VersionNegotiationExtEnabled    : 1;
     uint8_t RESERVED                        : 1;
     const uint32_t* DesiredVersionsList;
     uint32_t DesiredVersionsListLength;
+    uint16_t MinimumMtu;
```
Comment (anrossi) [inline docs/api/QUIC_SETTINGS.md:82]: I think if you arrange these from largest to smallest, you'll get better packing in the struct.

Comment (nibanks) [reply docs/api/QUIC_SETTINGS.md:82]: Yeah, but I don't want to change the previous settings.

Comment (ThadHouse) [reply docs/api/QUIC_SETTINGS.md:82]: Its a breaking change to rearrange them.

Comment (ThadHouse) [reply docs/api/QUIC_SETTINGS.md:82]: Also, these 2 are here because theres 4 bytes of padding after DesiredVersionsListLength.

Comment (anrossi) [reply docs/api/QUIC_SETTINGS.md:82]: I understand that. I mean among these new settings, you can rearrange them for better packing

Comment (anrossi) [reply docs/api/QUIC_SETTINGS.md:82]: oh, so it is the docs. My mistake.

**Review by ThadHouse — APPROVED** [review]

**Pull request merged**